### PR TITLE
Feat/email domain standalone

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/email-domain-module
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7169
| License          | BSD 3-Clause

## Description

email domain standalone application

- [x] TRANS-37983

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report translator-push#19896.0 ✘  
* 
  * push ✘

File: packages/manager/apps/email-domain/package.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@ovh-ux/manager-emaildomain-app",
+  "version": "0.3.0",
```
Comment (marie-j) [inline packages/manager/apps/email-domain/package.json:3]: Already been released ?

Comment (antleblanc) [reply packages/manager/apps/email-domain/package.json:3]: Good catch!

**Comment by antleblanc** [comment]
**question:** Assets folder couldn't be located into `src`?

From `packages/manager/apps/email-domain/assets/**` to `packages/manager/apps/email-domain/src/assets/**`.

**Review by zakariasahmane — APPROVED** [review]

**Comment by JeremyDec** [comment]
No translations for `DE/GB/ES/IT`?

**Review by antleblanc — CHANGES_REQUESTED** [review]
We tend to use `BREAKING CHANGE:` commit when it comes to apps/module scaffolding.

Could you please update following commit `10a8f9f2b44b591db196bd918cc6bbfd577d3e8a`?

Thank you!

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**